packages/react/src/__tests__/browser.test.tsx

```diff
@@ -37,7 +37,7 @@ describe('browser', () => {
     render(
       <>
         <StyledDiv>hello world</StyledDiv>
-        <StyledDiv>hello world</StyledDiv>
+        <StyledDiv>Hello World!</StyledDiv>
```

Comment:
I was having the tests in this file continue to fail unless I made any sort of change to the file. Coming back to this now I can see that it would be a Jest cache issue.
Running `npx jest --clearCache` now allows the test to pass locally with the original, so I'll undo this silly change.